compiler/src/iree/compiler/Codegen/Utils/CPUUtils.cpp

```diff
@@ -63,4 +66,28 @@ FailureOr<Operation *> getRootOperation(ArrayRef<Operation *> computeOps) {
   return rootOperation;
 }
 
+StringAttr getEnableLoopPeelingAttrName(MLIRContext *ctx) {
+  return StringAttr::get(ctx, kLoopPeelingAttrName);
+}
+
+bool isLoopPeelingEnabled(FunctionOpInterface funcOp) {
+  auto peelAttrName = getEnableLoopPeelingAttrName(funcOp->getContext());
+  auto trueAttr = BoolAttr::get(funcOp->getContext(), true);
+
+  IREE::Codegen::TranslationInfoAttr translationInfo =
+      getTranslationInfo(funcOp);
+  DictionaryAttr config = translationInfo.getConfiguration();
+
+  if (!config) {
+    return false;
+  }
+
+  auto peelAttr = translationInfo.getConfiguration().getNamed(peelAttrName);
+  if (peelAttr && peelAttr->getValue() == trueAttr) {
+    return true;
+  }
+
+  return false;
+}
```

Comment:
Not relevant if using a `UnitAttr`, but `BoolAttr::getValue()` returns a normal `bool` (there's no need to construct a `trueAttr` attribute). Also, the `if` is redundant here. 
```suggestion
  return peelAttr && peelAttr->getValue() == trueAttr;
}
```